_Classes/DeusEx/Classes/DeusExWeapon.uc

```diff
@@ -337,12 +337,13 @@ function bool DoRightFrob(DeusExPlayer frobber, bool objectInHand)
     return true;
 }
 
+// Trash: Sarge, is this thing obsolete now?
```

Comment:
The function isn't obsolete, but the last melee weapon code is (delete it, don't just comment it). Having a "draw" function may still be useful in the future, and it's good practice to allow event-driven programming through these sorts of functions. Just leave the function empty for now.